xbmc/pvr/channels/PVRChannelGroups.cpp

```diff
@@ -536,6 +536,11 @@ bool CPVRChannelGroups::DeleteGroup(const CPVRChannelGroup &group)
     CLog::Log(LOGERROR, "PVR - %s - cannot delete internal group '%s'", __FUNCTION__, group.GroupName().c_str());
     return false;
   }
+  if (group.GroupID() <= 0)
+  {
+    CLog::Log(LOGDEBUG, "PVR - %s - cannot delete group '%s' invalid id '%i'", __FUNCTION__, group.GroupName().c_str(), group.GroupID());
```

Comment:
I think we should keep it in the debug log for now after all the recent changes in the related code. We can always delete it later.

shouldn't we just return true here? as it's "deleted" because it doesn't exist in the db?